Library/Homebrew/os/mac/xcode.rb

```diff
@@ -141,9 +141,9 @@ def provides_cvs?
 
       def default_prefix?
         if version < "4.3"
-          %r{^/Developer} === prefix
+          prefix.start_with? "/Developer"
```

Comment:
This isn't a regression, but this will also match `/DeveloperTrolling/SomePath`. And it shouldn't!? (Some in the change below.)